src/missing-types.d.ts

```diff
@@ -52,12 +52,15 @@ declare module '@higlass/services' {
               bins_per_dimension?: number;
           }
     );
-    type TileData = {
+    export type TileData = Array<Record<string, string | number>> & {
```

Comment:
Yes, it is. It's an 'array-like object' for the sparse data, e.g., beddb.

<img width="1231" alt="Screen Shot 2022-08-17 at 12 04 48" src="https://user-images.githubusercontent.com/9922882/185188662-bced3118-f17c-40b8-b8a4-ac25520ee418.png">

Reply:
https://github.com/higlass/higlass/blob/38f0c4415f0595c3b9d685a754d6661dc9612f7c/app/scripts/TiledPixiTrack.js#L653

Reply:
Ah wow... In that case it might be good to have the tile typed with a discriminate union:

```typescript
interface TileDataBase {
	server: string;
	tileId: string;
	tilePos: number[];
	tilePositionId: string;
	tilesetUid: string;
	zoomLevel: number;
}

inferface DenseTileData extends TileDataBase {
	dense: number[];
}

type IntervalTile = { xStart: number, xEnd: number, chrOffset: number, importance: number, uid: string } 
type IntervalTileData = Array<IntervalTile> & TileDataBase;

type TileData = DenseTileData | IntervalTileData;
```